src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -110,6 +120,12 @@ class ActorWorkflowManager extends WorkflowManagerActor {
     } yield context.actorOf(WorkflowActor.buildWorkflowActorProps(id, binding, coercedInputs))
   }
 
+  override def notifyCompletion(obj: Option[AnyRef]): Unit = {
```

Comment:
I suppose a more message-passing actor-y approach is better, which would require the thread that's calling this, the SingleWorkflowRunner needs to be an actor?